packages/ui-components/src/components/PageHeader/PageHeader.component.tsx

```diff
@@ -4,8 +4,7 @@
  */
 
 import React from "react"
-import PropTypes from "prop-types"
-import SAPLogo from "../../img/sap_logo.svg"
+import { SapSvg } from "../../svg/SapSvg"
```

Comment:
Why the name change? The old name told me exactly what it is, the new name tells me that it's an SVG but I lose the semantic information.

Reply:
Good point. I will change it.